starboard/rdk/launcher.py

```diff
@@ -172,6 +172,14 @@ def _request_payload(method):
           f'"configuration":{{ {cmd_test_param}}} }} }}\' ')
       return json_cmd
 
+    # test file preparation
```

Comment:
You injected this block of code between a function and the block of code that uses it.

I think this would make more sense if it was moved up to before the rsync commands.

Reply:
This part of change is moved to a separate CL
 https://github.com/youtube/cobalt/pull/1697